src/server/routes/editor.js

```diff
@@ -155,28 +178,7 @@ router.get('/:id', (req, res, next) => {
 
 			return editorJSON;
 		})
-		.then((editorJSON) => {
-			let editorTitleJSON;
-			if (editorJSON.titleUnlockId === null) {
-				editorTitleJSON = Promise.resolve(editorJSON);
-			}
-			else {
-				editorTitleJSON = new TitleUnlock({
-					id: editorJSON.titleUnlockId
-				})
-					.fetch({
-						withRelated: ['title']
-					})
-					.then((unlock) => {
-						if (unlock !== null) {
-							editorJSON.title =
-								unlock.relations.title.attributes;
-						}
-						return editorJSON;
-					});
-			}
-			return editorTitleJSON;
-		})
+		.then(getEditorTitleJSON, TitleUnlock)
```

Comment:
What is the second parameter here for? The promise spec suggests `then` takes two functions.

Reply:
This was stupid of me - I wanted to pass it to `getEditorTitleJSON`.